modules/Microsoft.MachineLearningServices/workspaces/deploy.bicep

```diff
@@ -131,6 +133,12 @@ param imageBuildCompute string = ''
 @sys.description('Conditional. The user assigned identity resource ID that represents the workspace identity. Required if \'userAssignedIdentities\' is not empty and may not be used if \'systemAssignedIdentity\' is enabled.')
 param primaryUserAssignedIdentity string = ''
 
+@sys.description('Optional. The service managed resource settings.')
```

Comment:
Should these new parameters be added to the common tests?

Reply:
In theory yes - but I haven't had time to look into how they work. Just saw they were missing. I don't think it's too bad as we're just passing the value through without any logic - but if you think it should be required anyways I'll try to find time to figure out how I can get the PrivateLinks & CosmosDB in here.

Reply:
Let's postpone that outside of this PR, but let's create an issue for that. I'd expect we have the same for many modules, but maybe easier to at least track the ones we notice.

Reply:
Created #2247